jenkins/jk-setup.sh

```diff
@@ -133,12 +133,11 @@ elif [[ $COMPILER == *icc* ]]; then
   export CC=`which icc`
   export CXX=`which icc`
   export FC=`which ifort`
-  export ExtraCMakeOptions="${ExtraCMakeOptions} -Dvc=OFF"
```

Comment:
Yes, it should be fine to use icc to compile Vc. We also only have icc17 builds now, which are definitely fine for the versions of Vc we use.